lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, fig, rect=None, *args, **kwargs):
           *elev*             Elevation viewing angle (default 30)
           *zscale*           [%(scale)s]
           *sharez*           Other axes to share z-limits with
+          *persp*            Perspective projection (default True)
```

Comment:
Maybe use `projection` instead of `proj` to avoid breaking `get_proj`?

I think a better alternative would be to rename the current `get_proj` to `get_proj_matrix` - might be worth seeing what other devs think about that.